templates/artist/artist_list.html

```diff
@@ -24,11 +24,11 @@
                     {% endfor %}
                 </ul>
             </li>
-            <li><h5>artists in</h5></li>
+            <li><h6>artists in</h6></li>
             {% if genres %}
                 <li class="is-dropdown-submenu-parent opens-right">
-                    <a href="#"><h5>{{ active_genre }}</h5></a>
-                    <ul class="menu is-dropdown-submenu">
+                    <a href="#"><h6>{{ active_genre }}</h6></a>
+                    <ul class="menu is-dropdown-submenu>
```

Comment:
It looks like the close quote is missing after `is-dropdown-submenu`. I think that's what's preventing the first option from not being clickable.